File: test/test1/main.c
#include <stdio.h> 
#include <math.h> 
#include <main.h> 

int main(int *argc,char* *argv)
{
  int __retv;
  int xs[11];
  int i;
  int sum;
  sum = 0;
  i = 0;
  int __fbound0 = 10;
  for (; i <= __fbound0; ++i) {
    xs[i] = i;
    printf("%d\n",xs[i]);
    sum = sum + i;
  }
  printf("%s%d\n","and their sum is: ",sum);
  __retv = 0;
  return __retv;
}

File: test/test1/main.f90
integer(4) function main(argc, argv)

    integer(4) :: argc
    character(len=*), dimension(*) :: argv

    integer(4), dimension(0:10) :: xs
    integer(4) :: i, sum

    sum = 0

    do i = 0,10
       xs(i) = i
       print *, xs(i)
       sum = sum + i
    enddo

    print *, "and their sum is: ", sum

    main = 0
    return
end
